src/lesson04/MyLinkedList.java

```diff
@@ -0,0 +1,106 @@
+package lesson04;
+
+public class MyLinkedList<T> implements List<T> {
+    private int size = 0;
+    private Node<T> first;
+    private Node<T> last;
+
+    public MyLinkedList() {
+    }
+
+    @Override
+    public T get(int index) {
+        if (index < 0 && index >= size) {
+            System.out.println("Enter correct index");
+        }
+        return node(index).item;
+    }
+
+    @Override
+    public void add(T t) {
+        Node<T> currentNode = first;
+        if (currentNode == null) {
+            Node<T> last = this.last;
+            Node<T> newNode = new Node<>(last, t, null);
+            this.last = newNode;
+            if (last == null)
+                currentNode = newNode;
+            else
+                last.next = newNode;
+            size++;
```

Comment:
You can see size++ in if and else blocks. So it possible to remove line 30 and  line 35. And put size++ after if-else block.